### PR TITLE
Angular enhancements

### PR DESCRIPTION
**Pull request by zRosenthal**
Expose state as an observable to give more flexibility (leverage pipes)

Remove dependency on BrowserAnimationsModule as it should be treated the same as the BrowserModule and be required in the root of the angular app.

### CODE REVIEW

**Comment by sdirix** [comment]
Thanks for the contribution! :heart: We'll take a look soon!

File: packages/angular/src/jsonforms.service.ts [hunk 1]
```diff
@@ -19,11 +30,11 @@ export class JsonFormsAngularService {
         this.state = new BehaviorSubject({ jsonforms: this._state });
     }
 
-    subscribe(observer: PartialObserver<JsonFormsState>): Subscription {
+    get $state(): Observable<JsonFormsState> {
```
Comment (eneufeld) [inline packages/angular/src/jsonforms.service.ts:33]: I suggest to keep the subscribe method but mark it as deprecated.
@sdirix what do you think?

Comment (sdirix) [reply packages/angular/src/jsonforms.service.ts:33]: Besides some alpha releases this was not yet made public so I would be fine with changing this now without deprecation.

Comment (eneufeld) [reply packages/angular/src/jsonforms.service.ts:33]: agreed,

**Comment by zRosenthal** [comment]
@sdirix @eneufeld Thanks for taking a look at this!
Any idea when this might get merged/released?

**Comment by sdirix** [comment]
We'll release this (as an alpha release) at latest at some point next week. The regular release probably comes at some point next month.

**Comment by sdirix** [comment]
Thanks again for the PR :heart:

**Comment by sdirix** [comment]
@zRosenthal We managed to get the alpha release out today. `2.4.0-alpha.3` is ready to be consumed :+1:

**Pull request merged**